### PR TITLE
Reversion the API

### PR DESCRIPTION
**Pull request by rheinardkorf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,12 +28,12 @@ Both above authentication methods will return a JWT that contains the user's ID
...
 Example: Get token with User Authentication
 ```
-curl -X POST -F 'username=admin' -F 'password=password' http://local.dev/wp-json/tide/v2/auth
+curl -X POST -F 'username=admin' -F 'password=password' http://local.dev/wp-json/tide/v1/auth
 ```
 
 Example: Get token with API Key and API Secret
 ```
-curl -X POST -F 'api_key=1guJ0omfNkzoXsUu6NRp9rwCr' -F 'api_secret=X)WxT3Wp&NPHPWFFxRGDioP1UUVNuBfH' http://local.dev/wp-json/tide/v2/auth
+curl -X POST -F 'api_key=1guJ0omfNkzoXsUu6NRp9rwCr' -F 'api_secret=X)WxT3Wp&NPHPWFFxRGDioP1UUVNuBfH' http://local.dev/api/tide/v2/auth
```
Comment (valendesigns) [inline README.md]: Let's change `local.dev` to `[SITE]` like below.

Comment (valendesigns) [reply README.md]: @rheinardkorf Also this should be `v1` not `v2`.

Comment (rheinardkorf) [reply README.md]: good catch

**Comment by rheinardkorf** [comment]
@valendesigns all your's

File: README.md [hunk 2]
```diff
@@ -28,19 +28,19 @@ Both above authentication methods will return a JWT that contains the user's ID
 
 Example: Get token with User Authentication
 ```
-curl -X POST -F 'username=admin' -F 'password=password' http://local.dev/wp-json/tide/v2/auth
+curl -X POST -F 'username=admin' -F 'password=password' http://[SITE]/wp-json/tide/v1/auth
```
Comment (valendesigns) [inline README.md]: @rheinardkorf `wp-json` should be `api`, as well.

File: README.md [hunk 3]
```diff
@@ -28,19 +28,19 @@ Both above authentication methods will return a JWT that contains the user's ID
...
-curl -X POST -F 'api_key=1guJ0omfNkzoXsUu6NRp9rwCr' -F 'api_secret=X)WxT3Wp&NPHPWFFxRGDioP1UUVNuBfH' http://local.dev/wp-json/tide/v2/auth
+curl -X POST -F 'api_key=1guJ0omfNkzoXsUu6NRp9rwCr' -F 'api_secret=X)WxT3Wp&NPHPWFFxRGDioP1UUVNuBfH' http://[SITE]/api/tide/v1/auth
 ```
 
 Once a token is obtained you can access other REST API resources by specifying the HTTP Authorization header with a bearer token.
 
 Example: Standard WordPress endpoint request with JWT authentication active and using a Bearer token
 ```
-curl -H "Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE0ODc1NzAwOTIsImlzcyI6Imh0dHA6XC9cL3NpbmdsZTUuZGV2IiwiZXhwIjoxNDkwMTYyMDkyLCJkYXRhIjp7ImNsaWVudCI6eyJpZCI6MSwidHlwZSI6IndwX3VzZXIifX19.HGqNFaH742QPsMy0zkFmuiiRvlBczVoeorr6uVnwwZ4" http://local.dev/wp-json/wp/v2/posts
+curl -H "Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE0ODc1NzAwOTIsImlzcyI6Imh0dHA6XC9cL3NpbmdsZTUuZGV2IiwiZXhwIjoxNDkwMTYyMDkyLCJkYXRhIjp7ImNsaWVudCI6eyJpZCI6MSwidHlwZSI6IndwX3VzZXIifX19.HGqNFaH742QPsMy0zkFmuiiRvlBczVoeorr6uVnwwZ4" http://[SITE]/wp-json/wp/v1/posts
```
Comment (valendesigns) [inline README.md]: Same here, we should be using `api` not `wp-json`.

File: README.md [hunk 4]
```diff
@@ -61,10 +61,10 @@ that software clients use "Refresh Tokens" to authenticate when possible to avoi
...
 Format:
 ```
-curl -X POST -H "Authorization: Bearer [REFRESH_TOKEN]" "https://[SITE]/api/tide/v2/auth"
+curl -X POST -H "Authorization: Bearer [REFRESH_TOKEN]" "https://[SITE]/api/tide/v1/auth"
 ```
 
 Example: Re-authenticate with a "Refresh Token"
 ```
-curl -X POST -H "Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE0ODgxNjYwNjQsImlzcyI6Imh0dHA6XC9cL3NpbmdsZTUuZGV2IiwiZXhwIjoxNTE5NzAyMDY0LCJkYXRhIjp7InRva2VuX3R5cGUiOiJyZWZyZXNoIiwiY2xpZW50Ijp7ImlkIjoxLCJ0eXBlIjoid3BfdXNlciJ9fX0.FP11UCDo-5AiYKacL545tPwgsEQUYwMXkqapqoPVYuw" "http://example.dev/wp-json/tide/v2/auth"
+curl -X POST -H "Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE0ODgxNjYwNjQsImlzcyI6Imh0dHA6XC9cL3NpbmdsZTUuZGV2IiwiZXhwIjoxNTE5NzAyMDY0LCJkYXRhIjp7InRva2VuX3R5cGUiOiJyZWZyZXNoIiwiY2xpZW50Ijp7ImlkIjoxLCJ0eXBlIjoid3BfdXNlciJ9fX0.FP11UCDo-5AiYKacL545tPwgsEQUYwMXkqapqoPVYuw" "http://example.dev/wp-json/tide/v1/auth"
```
Comment (valendesigns) [inline README.md]: Another instance of `wp-json`.

**Comment by rheinardkorf** [comment]
@valendesigns hopefully caught them all.

**Comment by valendesigns** [comment]
@rheinardkorf Did you run `npm install` again or something, the `package-lock.json` was updated for some reason.

**Comment by rheinardkorf** [comment]
@valendesigns I was wondering about that because I didn't run `npm` or `yarn` at all. Could adding the `dev-lib` pre-commit hook do something? or some weird actions by `composer install` (which I did do)

**Comment by valendesigns** [comment]
@rheinardkorf `composer install` should have only read the lock for composer and installed the dependencies. It is pretty odd that the file was changed somehow when you didn't run `npm` but potentially the `dev-lib` could have done it though it didn't happen to me so not sure.

**Comment by valendesigns** [comment]
Honestly, I'm not sure why we even have `package.json`. IMO it should be removed.

**Comment by valendesigns** [comment]
Which also leads me down the why are there empty CSS & JS files and the admin page that does nothing. We should remove all this extra stuff we don't need or use.

**Pull request merged**